apps/api/src/view-models/buildViewModelForEntity/viewModelBuilders/buildTermViewModels.ts

```diff
@@ -1,23 +1,37 @@
-import { Term } from 'apps/api/src/domain/models/term/entities/term.entity'
-import { entityTypes } from 'apps/api/src/domain/types/entityType'
-import { isInternalError } from 'apps/api/src/lib/errors/InternalError'
-import { ViewModelBuilderDependencies } from '../buildViewModelForEntity'
-import { TermViewModel } from '../viewModels'
+import { Term } from 'apps/api/src/domain/models/term/entities/term.entity';
+import IsPublished from 'apps/api/src/domain/repositories/specifications/isPublished';
+import { entityTypes } from 'apps/api/src/domain/types/entityTypes';
+import { isInternalError } from 'apps/api/src/lib/errors/InternalError';
+import { TermViewModel } from '../viewModels';
+import { ViewModelBuilderDependencies } from './types/ViewModelBuilderDependencies';
 
-export default async ({
-    repositoryProvider,
-    configService,
-}: ViewModelBuilderDependencies): Promise<TermViewModel[]> => {
-    const termRepository = repositoryProvider.forEntity<Term>(entityTypes.term)
+type Options = {
```

Comment:
Do the same for the other view model builders.

Reply:
done